mypy/test/helpers.py

```diff
@@ -350,11 +352,12 @@ def parse_options(
         if "--show-error-codes" not in flag_list:
             options.hide_error_codes = True
     else:
-        flag_list = []
         options = Options()
+        options.pretty = False
         options.error_summary = False
         options.hide_error_codes = True
         options.force_union_syntax = True
+        flag_list = ["--no-pretty"]
```

Comment:
Do you really have to set `flag_list = ["--no-pretty"]` in addition to `options.pretty = False`? Weird.